int32 drone_id
int32[] measured_ids
geometry_msgs/Vector3[] bearingVectors